src/bigquery.ts

```diff
@@ -2746,6 +2737,9 @@ export class BigQueryDatetime {
       time = time ? ' ' + time : '';
       value = `${y}-${m}-${d}${time}`;
     } else {
+      if (value.length > 100) {
+        throw new Error('BigQueryDatetime input too long');
+      }
```

Comment:
More context needed! Why is this here? Is this related to the query stuff? If it needs to be in this PR, please edit the message to specify that it needs to be 100 characters or fewer and consider a link to our docs

Reply:
this was to solve a security issue, I'll move that to a separated PR, sorry about that.